src/test/java/Page_Objects/InputSelectionPage.java

```diff
@@ -21,7 +21,7 @@ public class InputSelectionPage {
     @FindBy(css = "button[data-test='advanced']")
     private WebElement advancedSetupButton;
 
-    @FindBy(css = "button[data-test='skip']")
+    @FindBy(xpath = "//button[@data-test='skip']")

```

Comment:
I believe that best practice is to not specify the element type (`button`, `div` etc.) if you don't have to (e.g. you have a much more specific `data-test` attribute to use). Otherwise the tests are more likely to break with changes in the app. And it doesn't make a difference for end users if an element is a `button` or `a`, so neither it should for the tests (if they don't have to)

The reason is that these types can be changed in the code and the developer wouldn't be aware that it'd break the test. With `data-test` attribute, they'd be much more likely to expect that changing it would break something (and they usually don't change it).